modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -9,8 +9,8 @@ class AppointmentsController < ApplicationController
 
       def index
         use_cache = params[:useCache] || true
-        start_date = params[:startDate] || beginning_of_last_year.iso8601
-        end_date = params[:endDate] || one_year_from_now.iso8601
+        start_date = params[:startDate] || appointments_cache_interface.latest_allowable_cache_start_date.iso8601
```

Comment:
I question whether we should be doing this fallback to the cache default dates. The only other use for this date is for pagination params, and I'm not sure I see the value of setting these in the pagination params. And having them complicates the code. To make this work, I had to publicly expose these methods on the interface and make some other small changes. But changing this behavior breaks a lot of specs and it didn't seem worth mucking around with.